### DISCUSSION TITLE
Join us in the Astroid Framework Facebook group

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#505] [Announcements]
Hello my friends,

I appreciate you being a part of the Astroid family. So that we can easily exchange and develop the community, I have launched the Astroid community group on the social network Facebook. I look forward to getting to know each other, exchange and share with you about Astroid and other issues on the group. It's a pleasure to welcome all of you. :)

You can request to join group via url: https://www.facebook.com/groups/astroid
Or scan this QR Code
![410795999_1027385391822459_8038182486971426416_n](https://github.com/templaza/astroid-framework/assets/3204103/6e336796-9cf9-4ff9-8522-f66c0e440187)

Thank you & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Membership  pending. Good Day to the great Astroid Community.
**THE BEST FRAMEWORK FOR JOOMLA!**

**Reply by nidzo80** [reply]
With pleasure!
